pandas/core/series.py

```diff
@@ -2573,6 +2573,9 @@ def quantile(
 
                 * linear: `i + (j - i) * fraction`, where `fraction` is the
                   fractional part of the index surrounded by `i` and `j`.
+                  For example, if (len(self) - 1) * q is 9.6,
```

Comment:
I don't think this example clarifies the last sentence much. I think `fractional part of the index surrounded by` should use clearer language